config/fxdata/magic.cfg

```diff
@@ -1126,8 +1126,8 @@ Name = SHOT_DIG
 Animation = 0
 AnimationSize = 0
 AnimationTransparency = 0
-Size_XY = 128
-Size_Z = 128
+Size_XY = 1
+Size_Z = 1
```

Comment:
No, I could still reproduce it at 64, and I think even 32. Would it even be able to hit anything at 0?

Reply:
> > No, I could still reproduce it at 64, and I think even 32. Would it even be able to hit anything at 0?
> 
> Mh you are right it doesn't hit anything at 0 on possession but it does outside of it. 🤔
>

That's because digging in possession it completely separate from digging outside it. Digging outside possession is it own instance, which doesn't involve this shot at all.